packages/emotion-primitives/test/emotion-primitives.test.js

```diff
@@ -21,6 +23,16 @@ describe('Emotion primitives', () => {
     )
   })
 
+  test('compose via .styles prop', () => {
+    const Text = emotionPrimitive.text`color: hotpink;`
+
+    const Name = emotionPrimitive.text`${Text.styles}`
```

Comment:
For composition, I think we have to rely on `.styles` static property of the component. I don't any other way to tackle this. But still, open to discuss about it.